docs/guides/developer/docs/api/events-api.md

```diff
@@ -886,9 +886,10 @@ Available since API version 1.1.0.
 
 Update the scheduling information of the event with id `{event_id}`.
 
-Form Parameters             |Type            | Description
-:---------------------------|:---------------|:----------------------------
-`scheduling`                | `string`       | The scheduling information.
+Form Parameters             |Type            | Description                              | Default | Version
+:---------------------------|:---------------|:-----------------------------------------|:--------|:-------
+`scheduling`                | `string`       | The scheduling information.              | <span class="required">Required</span>| 1.1.0
```

Comment:
Should we really use HTML in markdown? Seems somewhat superfluous here.

Reply:
The styling of "Required" text adds value to this case where you can immediately see which field is actually required. It enriches the documentation. With markdown you can't really control how it will look.

The style can be reused to update the rest of the documentation later on.

Reply:
Ok.